components/pricing.tsx

```diff
@@ -0,0 +1,156 @@
+          {buttonText}
+        </Button>
+      )}
+    </CardContent>
+    <CardFooter className="p-6">
+      {!isFree && features && (
+        <ul className="flex-grow space-y-4">
+          {features.map((feature, index) => (
+            <li key={index} className="flex items-center">
+              <Check className="mr-3 h-5 w-5 flex-shrink-0 text-primary-700" />
+              <span className="text-sm font-medium text-primary-700 sm:text-base">
+                {feature}
+              </span>
+            </li>
+          ))}
+        </ul>
+      )}
+    </CardFooter>
+  </Card>
+);
+
+const PricingPage: React.FC = () => {
+  const tiers: PricingTierProps[] = [
+    {
+      title: "Free",
+      price: "0",
+      description:
+        "Download PearAI and use it for free with your own API keys!",
+      isFree: true,
+    },
+    {
+      title: "Monthly",
+      price: "18",
+      description:
+        "Get the monthly subscription and get your supercharged coding editor assistant",
+      features: [
+        "Pro two-week trial",
+        "Unlimited Copilot++ completions",
+        "OpenAI zero-data retention",
+        "Unlimited Copilot++ completions",
+      ],
+      buttonText: "Get Started",
+    },
+    {
+      title: "Yearly",
+      price: "14",
+      description: "Pay yearly and get... add text here",
+      features: [
+        "Pro two-week trial",
+        "Unlimited Copilot++ completions",
+        "OpenAI zero-data retention",
+        "Unlimited Copilot++ completions",
+      ],
+      buttonText: "Get Started",
+    },
+  ];
+
+  return (
+    <section className="relative py-8 sm:py-12 md:py-16 lg:py-24">
+      <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+        <div className="space-y-12 sm:space-y-16 md:space-y-20 lg:space-y-24">
+          {/* Page header */}
+          <div className="mx-auto mt-16 max-w-4xl space-y-4 text-center sm:mt-0 sm:space-y-6">
+            <h1 className="text-3xl font-bold leading-tight sm:text-4xl md:text-5xl lg:text-6xl">
+              Fair pricing, unfair advantage.
+            </h1>
+            <p className="text-base font-medium text-gray-400 sm:text-lg md:text-xl lg:text-2xl">
+              Download PearAI today and get your coding efficiency to a new
+              level
+            </p>
+          </div>
+          {/* Pricing tiers */}
+          <div className="grid grid-cols-1 gap-8 sm:gap-10 md:grid-cols-2 lg:grid-cols-3">
+            {tiers.map((tier, index) => (
+              <div key={index} className="w-full">
+                <PricingTier {...tier} />
+              </div>
+            ))}
+          </div>
+          {/* Footer */}
+          <div className="text-center">
+            <p className="text-base text-gray-400 sm:text-lg md:text-xl">
+              Want to use it on a professional level?
+              <Link
+                href="/contact"
```

Comment:
this should also be the pearai support/business email and not redirect to /contact

Reply:
for now we can just do this instead of redirecting to a non-existing page
```suggestion
                href="#"
```